src/scripts/enable_ntp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+## Motivation:
+##    We have seen that clients might lose sync if the clock is not properly synchronized
+## Reference:
+##    https://wiki.debian.org/NTP
+
+# Install ntp package
+apt-get update
+apt-get install -y ntp
+
+# Enable on boot and restart ntp service
+sudo systemctl enable ntp
+sudo systemctl restart ntp
```

Comment:
Is there any posibility to only restart the ntp service to those having the issue?

Reply:
I'll look into it